packages/@stimulus/core/src/target_properties.ts

```diff
@@ -45,3 +55,7 @@ function defineLinkedProperties(object: any, properties: PropertyDescriptorMap)
     }
   })
 }
+
+function capitalize(name: string) {
+  return name.charAt(0).toLocaleUpperCase() + name.slice(1)
```

Comment:
Any risk of chopping a multibyte char here?

Reply:
Maybe…? It’d have to be a character outside the BMP which also has a corresponding capital. Seems unlikely enough that this will suffice for now.